handlers/geolocation/geolocation.go

```diff
@@ -220,7 +257,7 @@ func (c *GeolocationHandlersCollection) HandleStreamSource(ctx context.Context,
 			// stream is active, but STREAM_SOURCE cannot be found
 			glog.Errorf("error querying mist for active stream STREAM_SOURCE: %s", errMist)
```

Comment:
While we're in here are there any logging changes that would help us debug the root cause here? e.g adding a stream ID to this log line

Reply:
I improved logging in: 92f4a20df2fc4cd0514ffbb55e23462bab6781b8